src/containers/EmployeeList/Employees/index.tsx

```diff
@@ -13,42 +13,30 @@ export default function Employees({
   listEmployees,
   isValidating,
   pageNumber,
-}: employeesProps) {
+}: EmployeesProps) {
   const [employeeList, setEmployeeList] = useState(listEmployees);
   const [activePage, setPage] = useState(1);
-  const [search, setSearch] = useState('');
-  const [gender, setGender] = useState('all');
-  const [arrange, setArrange] = useState('all');
+  const [gender, setGender] = useState<string | null>('all');
+  const [arrange, setArrange] = useState<string | null>('all');
```

Comment:
Typing in here is redundant. Because state of `useState` always has value (as it a select box) and typescript is smart enough to know it's a string.

Reply:
![image](https://user-images.githubusercontent.com/72869134/166089401-8a5ce7f4-6b74-4ef2-931c-5ab767d0e7fb.png)
when i try to delete type

Reply:
Okay. I forgot to check type of value in `Select props`